### PR TITLE
Enable bookkeeper table service in pulsar standalone

### PR DESCRIPTION
**Pull request by sijie**
This change is enabling bookkeeper table service component in pulsar standalone. So people can try out stateful functions in pulsar.

### CODE REVIEW

**Comment by sijie** [comment]
@srkukarni

**Comment by sijie** [comment]
will update the bk version once 4.7.1 is done.

File: pulsar-zookeeper-utils/src/main/java/org/apache/pulsar/zookeeper/LocalBookkeeperEnsemble.java [hunk 1]
```diff
@@ -100,6 +114,10 @@ public LocalBookkeeperEnsemble(int numberOfBookies, int zkPort, int bkBasePort,
     ServerConfiguration bsConfs[];
     Integer initialPort = 5000;
 
+    // Stream/Table Storage
+    StreamStorageLifecycleComponent streamStorage;
+    Integer streamStoragePort = 4181;
```
Comment (nkurihar) [inline pulsar-zookeeper-utils/src/main/java/org/apache/pulsar/zookeeper/LocalBookkeeperEnsemble.java]: To make this value configurable seems better.

Comment (sijie) [reply pulsar-zookeeper-utils/src/main/java/org/apache/pulsar/zookeeper/LocalBookkeeperEnsemble.java]: yeah good point, will address that

**Comment by sijie** [comment]
4.7.1 rc0 is pushed to apache staging. so this PR is compilable now. 

I will address the remaining comments today, and it should be ready to go after that.

**Comment by sijie** [comment]
> so this PR is compilable now.

take my words back. I think we need to wait until 4.7.1 becomes available at maven central.

**Comment by sijie** [comment]
retest this please

File: pulsar-zookeeper-utils/src/main/java/org/apache/pulsar/zookeeper/LocalBookkeeperEnsemble.java (hunk 1 above)
Comment (sijie) [reply pulsar-zookeeper-utils/src/main/java/org/apache/pulsar/zookeeper/LocalBookkeeperEnsemble.java]: make it configurable from the constructor.

**Comment by sijie** [comment]
retest this please

File: pulsar-zookeeper-utils/src/main/java/org/apache/pulsar/zookeeper/LocalBookkeeperEnsemble.java (hunk 1 above)
Comment (nkurihar) [reply pulsar-zookeeper-utils/src/main/java/org/apache/pulsar/zookeeper/LocalBookkeeperEnsemble.java]: Thanks!
Should we add an option like
```
@Parameter(names = { "--stream-storage-port" }, description = "Local bookies stream storage port")
```
to `PulsarStandaloneStarter`?

Comment (sijie) [reply pulsar-zookeeper-utils/src/main/java/org/apache/pulsar/zookeeper/LocalBookkeeperEnsemble.java]: nice point. I added a starter parameter.

**Comment by sijie** [comment]
retest this please

**Comment by sijie** [comment]
CI passed. @merlimat @nkurihar can you review it?

File: pulsar-zookeeper-utils/src/main/java/org/apache/pulsar/zookeeper/LocalBookkeeperEnsemble.java (hunk 1 above)
Comment (nkurihar) [reply pulsar-zookeeper-utils/src/main/java/org/apache/pulsar/zookeeper/LocalBookkeeperEnsemble.java]: Thanks! 👍

**Pull request merged**